test/nools/task-emitter.spec.js

```diff
@@ -287,6 +306,25 @@ describe('task-emitter', () => {
         });
       });
 
+      it('resolvedIf is not required', () => {
```

Comment:
shouldn't have two tests with the same name. what's the difference?

Reply:
One was under "person based task" and another under "report based task", but that might be unnecessary, so removed one